math/eratosthenes.cpp

```diff
@@ -75,44 +88,14 @@ static void test() {
     assert(primes[8] == false);
     assert(primes[9] == false);
 
-    std::cout << "All tests have successfully passed!\n";
```

Comment:
\n is apparently better than std::endl it increases performance slightly but this is also good

Reply:
yea, its not worth changing in this one, but i'll do \n in the future!